sms/verify-signed-sms/.env.dist

```diff
@@ -1,7 +1,7 @@
 FLASK_APP = "app.py"
 FLASK_ENV = "development"
 FLASK_RUN_PORT = 3000
-FLASK_RUN_HOST = "0.0.0.0"
+# FLASK_RUN_HOST = "0.0.0.0" Uncomment to make the server publicly available
 
 VONAGE_API_KEY=
 VONAGE_SIGNATURE_SECRET=
```

Comment:
Do these three environment variables need values, or do they even need to be in this file? (e.g. `VONAGE_SIGNATURE_SECRET_METHOD=` is already defined in the top-level `.env.dist` file. Although I guess they're purposely left blank for the instruction:

> Copy `.env.dist` to `.env` and fill in your credentials

in the `README`?

Reply:
I've removed this entirely now, if you run with Ngrok as we recommend it's not needed.

Reply:
Have changed the file to be `.flaskenv` to allow us to use the top level `.env` values as well as the Flask environment variables.